src/rpc/mempool.cpp

```diff
@@ -255,7 +255,7 @@ static std::vector<RPCResult> MempoolEntryDescription()
             {RPCResult{RPCResult::Type::STR_HEX, "transactionid", "parent transaction id"}}},
         RPCResult{RPCResult::Type::ARR, "spentby", "unconfirmed transactions spending outputs from this transaction",
             {RPCResult{RPCResult::Type::STR_HEX, "transactionid", "child transaction id"}}},
-        RPCResult{RPCResult::Type::BOOL, "bip125-replaceable", "Whether this transaction could be replaced due to BIP125 (replace-by-fee)"},
+        RPCResult{RPCResult::Type::BOOL, "bip125-replaceable", "Whether this transaction signals BIP125 replaceability or has an unconfirmed ancestor signaling BIP125 replaceability.\n"},
```

Comment:
For reference, this change adds clarification around the  #22209 misunderstanding

Reply:
Yeah, hopefully this can close that.

This comment was helpful background for me: https://github.com/bitcoin/bitcoin/pull/25575#issuecomment-1179519204

Reply:
Hmm, depending on the user there might still be a need for just finding out the signal of this tx, not taking into account the ancestors.

So I think we could return two bools, or an enum, see https://github.com/bitcoin/bitcoin/pull/25575#issuecomment-1180079174

Reply:
> So I think we could return two bools, or an enum, see https://github.com/bitcoin/bitcoin/pull/25575#issuecomment-1180079174

either would make sense imo :shrug: